bigtable/tests/system.py

```diff
@@ -238,6 +244,93 @@ def test_create_app_profile_with_single_routing_policy(self):
+
+        cluster.reload()
+        self.assertEqual(cluster.location_id, LOCATION_ID)
+        self.assertEqual(cluster.state, ClusterState.READY)
+        self.assertEqual(cluster.serve_nodes, SERVE_NODES)
+        # Make sure that by default an StorageType.SSD storage is used.
+        self.assertEqual(cluster.default_storage_type, StorageType.SSD)
+
+    def test_update_cluster(self):
+        NEW_SERVE_NODES = 7
+
+        # TODO: After instance.create can accept clusters this should just
+        # be replaced with Config.CLUSTER (from `setUpModule`)
+        cluster = Config.INSTANCE.cluster(CLUSTER_ID)
+        # Get current cluster metadata
+        cluster.reload()
+
+        self.assertEqual(cluster.serve_nodes, SERVE_NODES)
+        cluster.serve_nodes = NEW_SERVE_NODES
+        operation = cluster.update()
+
+        # We want to make sure the operation completes.
+        operation.result(timeout=10)
+
+        # Create a new cluster instance and reload it.
+        alt_cluster = Config.INSTANCE.cluster(CLUSTER_ID)
+        alt_cluster.reload()
+        self.assertEqual(alt_cluster.serve_nodes, NEW_SERVE_NODES)
+
+        # Make sure to put the cluster back the way it was for the
+        # other test cases.
+        # TODO: After instance.create can accept clusters this should
+        # be replaced with Config.CLUSTER (from `setUpModule`)
+        cluster.serve_nodes = SERVE_NODES
+        operation = cluster.update()
+        operation.result(timeout=10)
+
+    def test_create_cluster(self):
+        from google.cloud.bigtable.enums import StorageType
+        from google.cloud.bigtable.enums import ClusterState
+
+        ALT_CLUSTER_ID = INSTANCE_ID+'-cluster2'
+        ALT_LOCATION_ID = 'us-central1-f'
+        ALT_SERVE_NODES = 5
+
+        cluster = Config.INSTANCE.cluster(ALT_CLUSTER_ID,
+                                          location_id=ALT_LOCATION_ID,
+                                          serve_nodes=ALT_SERVE_NODES,
+                                          default_storage_type=StorageType.SSD)
+        operation = cluster.create()
+
+        # We want to make sure the operation completes.
+        operation.result(timeout=10)
+
+        # Create a new object instance and make sure it is the same.
+        alt_cluster = Config.INSTANCE.cluster(ALT_CLUSTER_ID)
+        alt_cluster.reload()
+
+        self.assertEqual(cluster, alt_cluster)
+        self.assertEqual(cluster.location_id, alt_cluster.location_id)
+        self.assertEqual(alt_cluster.state, ClusterState.READY)
+        self.assertEqual(cluster.serve_nodes, alt_cluster.serve_nodes)
+        self.assertEqual(cluster.default_storage_type,
+                         alt_cluster.default_storage_type)
+
+        # Delete the newly created cluster and confirm
+        self.assertTrue(self._cluster_exists(cluster.name))
+        cluster.delete()
+        self.assertFalse(self._cluster_exists(cluster.name))
+
+    def _cluster_exists(self, cluster_name):
```

Comment:
Can you please create a `Cluster.exists()` method?  That seems useful.

Reply:
Done.
Added unit test and system test.